src/components/navigation-bar/navigation-bar.stories.js

```diff
@@ -1,47 +1,37 @@
 import React from 'react';
-import { storiesOf } from '@storybook/react';
-import { text, select } from '@storybook/addon-knobs';
-import { dlsThemeSelector, classicThemeSelector } from '../../../.storybook/theme-selectors';
-import NavigationBar from './navigation-bar';
-import notes from './notes.md';
-import './navigation-bar.stories.scss';
-import getDocGenInfo from '../../utils/helpers/docgen-info';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
+import NavigationBar from './navigation-bar.component';
 
-NavigationBar.__docgenInfo = getDocGenInfo(
-  require('./docgenInfo.json'),
-  /navigation-bar\.js(?!spec)/
-);
+export default {
+  title: 'Test/NavigationBar',
+  component: NavigationBar,
+  decorators: [withKnobs],
+  parameters: {
+    info: {
+      disable: true
+    },
+    knobs: { escapeHTML: false },
+    chromatic: {
+      disable: true
+    }
+  }
+};
 
-function makeStory(name, themeSelector, disableChromatic = false) {
-  const component = () => {
-    const children = text('children', '');
-    const as = select(
-      'as',
-      ['primary', 'secondary', 'transparent'],
-      NavigationBar.defaultProps.as
-    );
+export const Basic = () => {
+  const children = text('children', 'Example content');
+  const navigationType = select('navigationType', ['light', 'dark'], NavigationBar.defaultProps.navigationType);
```

Comment:
There is no need to pass here those props, we have already rendered those variants in `mdx` and making `chromatic` runs.